app/services/fraud_review_check.rb

```diff
@@ -0,0 +1,29 @@
+class FraudReviewCheck
```

Comment:
WDYT of naming this with `-er` as in `FraudReviewChecker`? I like services names to hint at what verbs they have when possible